### PR TITLE
`signalr` - `docs example improvement`

### PR DESCRIPTION
**Pull request by shu-ying789**

### CODE REVIEW

File: website/docs/r/web_pubsub_hub.html.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ resource "azurerm_user_assigned_identity" "test" {
   location            = azurerm_resource_group.example.location
 }
 
-resource "azurerm_web_pubsub" "example" {
+resource "azurerm_web_pubsub" "test" {
```
Comment (katbyte) [inline website/docs/r/web_pubsub_hub.html.markdown:27]: we use example in docs, so could we change this back?

File: website/docs/r/web_pubsub_hub.html.markdown [hunk 2]
```diff
@@ -34,8 +34,8 @@ resource "azurerm_web_pubsub" "example" {
 }
 
 resource "azurerm_web_pubsub_hub" "test" {
```
Comment (katbyte) [inline website/docs/r/web_pubsub_hub.html.markdown:36]:
```suggestion
resource "azurerm_web_pubsub_hub" "example" {
```

File: website/docs/r/web_pubsub_hub.html.markdown [hunk 3]
```diff
@@ -34,8 +34,8 @@ resource "azurerm_web_pubsub" "example" {
...
-  name          = "tfex-wpsh"
-  web_pubsub_id = azurerm_web_pubsub.exmaple.id
+  name          = "tfex_wpsh"
+  web_pubsub_id = azurerm_web_pubsub.test.id
```
Comment (katbyte) [inline website/docs/r/web_pubsub_hub.html.markdown:38]: ect

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/web_pubsub_hub.html.markdown (hunk 1 above)
Comment (shu-ying789) [reply website/docs/r/web_pubsub_hub.html.markdown]: @katbyte I have modified it, please check it, thanks.

**Review by katbyte — APPROVED** [review]
Thanks @shu-ying789 - LGTM 🌟

**Pull request merged**